src/utils/styles.js

```diff
@@ -5,3 +5,8 @@ export const colors = {
   darkGrey: "#4D4E51",
   white: "#FFFFFF",
 }
+
+export const borderRadius = {
+  small: "5px", // 0.3125em
+  medium: "8px", // 0.5em
```

Comment:
A question - why you decided to go with the px-units here? Asking just to learn more 😊

Reply:
Sorry, I was supposed to change these to em's too! Will do it now. (I'm used using pixels, so I coded the changes using pixels and then converted them to em's, but obviously forgot to change these 😬 )

Reply:
Ah, that's understandable!